frontend/src/metabase/admin/app/actions.ts

```diff
@@ -1,6 +1,5 @@
-import { createAsyncThunk } from "@reduxjs/toolkit";
-
 import { updateSetting } from "metabase/admin/settings/settings";
+import { createAsyncThunk } from "metabase/lib/redux";
```

Comment:
`createAsyncThunk` from `"@reduxjs/toolkit"` is not typed with our types, we should probably create a rule to not import it directly (I thought we already had it actually)